duckduckgo/iOS
# content blocking part 2

Reviewer: Mia
Asana: https://app.asana.com/0/0/427251562039377/f
CC:

**Description**:

This PR adds easylist blocking and addresses issues raised in the last PR.

Note that whitelisting and first party url logic is coming in the 3rd and final part of this trilogy. 

**Steps to test this PR**:
1. Access a site using trackers that will be caught by easy list (and not disconnect me).
2. Observe the updated site grade
3. Observe easylist tracker detection messages


###### Reviewer Checklist:


###### PR DRI Checklist:
